File: C# Fundamentals/03. Operators and Expressions/01, Odd or Even/01.Odd or Even.cs
using System;

namespace _01__Odd_or_Even
{
    class Program
    {
        static void Main()
        {
            int number = int.Parse(Console.ReadLine());

            Console.WriteLine((number%2==0)? "even " + number : "odd " + number);
        }
    }
}

File: 03. JavaScript Fundamentals/04. Operators and Expressions/01.Odd or Even.js
function solve(args) {
    var number = +args[0];
    var oddOrEven = null;
    if(number % 2 == 0) {
        oddOrEven = "even";
    }
    else oddOrEven = "odd";
    console.log(oddOrEven + " " + number);
}